tests/unit/test_project.py

```diff
@@ -140,7 +153,7 @@ def test_unmarshal_no_repositories(yaml_loaded_data):
 
     project = Project.unmarshal(yaml_loaded_data)
 
-    assert project.package_repositories == []
+    assert project.package_repositories is None
```

Comment:
Do we care about the specific value, or would `assert not project.package_repositories` be sufficient?

Reply:
eh I guess I don't have a personal preference here